test/test_utils/test_sparql_queries.py

```diff
@@ -0,0 +1,110 @@
+from src.pandasaurus.utils.sparql_queries import (
+    get_contextual_enrichment_query,
+    get_curie_prefix_validation_query,
+    get_full_enrichment_query,
+    get_label_query,
+    get_minimal_enrichment_query,
+    get_obsolete_term_query,
+    get_replaced_by_query,
+    get_simple_enrichment_query,
+    get_slim_list_query,
+    get_slim_members_query,
+)
+
+
+def test_get_simple_enrichment_query():
+    s_iri_list = ["CL:0000084", "CL:0000787"]
+    o_iri_list = ["CL:0000788", "CL:0000798"]
+    property_list = ["rdfs:subClassOf"]
+
+    query = get_simple_enrichment_query(s_iri_list, o_iri_list, property_list)
+
+    expected_query = (
+        "SELECT ?s ?s_label ?p ?o ?o_label WHERE { GRAPH <http://reasoner.renci.org/redundant> {"
+        "VALUES ?s { CL:0000084 CL:0000787 } VALUES ?o { CL:0000788 CL:0000798 } VALUES ?p { rdfs:subClassOf }"
+        "?s ?p ?o. } ?s rdfs:label ?s_label. ?o rdfs:label ?o_label. FILTER(?s != ?o)}# LIMIT"
+    )
+
+    assert query == expected_query
+
+
+def test_get_minimal_enrichment_query():
+    pass
+
+
+def test_get_full_enrichment_query():
+    s_iri_list = ["CL:0000084", "CL:0000787"]
```

Comment:
should we test with invalid CURIEs `CL_0000084` and unknown prefixes `NONE:0000084`